### PR TITLE
Restaurant Page

### PR DESCRIPTION
**Pull request by Godswilly**
- Created a restaurant site using Javascript DOM 
- I made use of import and export to call objects
- I Implemented ES6 functions

### CODE REVIEW

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes Required

### Hi @Godswilly  :wave: 

There are some updates to make in your project.

* [ ] Your package.json file is missing a a package: **webpack-dev-server**
* [ ] When I tried to start your application with **npm start**, I had this error below:

                Error: Cannot find module 'webpack-cli/bin/config-yargs' 

#### That's it, rectify the error and update your package.json file; then submit another form for a code review.

### Have a great day :slightly_smiling_face: 

### ``` Uchechukwu Henry Inyama```

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Restaurant page</title>
+</head>
+<body>
+  <div id="content">
```
Comment (peterrobert) [inline dist/index.html:9]: You have left out  a closing tag for this <div>

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED ♻️ 🛑 
Hello, @Godswilly  very nice project. I love the way you have modularised your code. Your work is very impressive 👍🏿 . However, there are a  few changes to be made. Kindly check below ⬇️ 

- [ ]  Files not related to the project example .vscode should be added to the .gitignore and should not be committed. 
- [ ] In your readme file, you have added `npm run watch` this is not configured in your package .json file. Kindly do so.

- [ ]  After running eslint on my machine, there appear to be some errors ⬇️ 
Screenshot from 2021-01-28 06-58-53

kindly fix these errors.

### if you have any questions kindly reach me on slack @peterrobert

### HAPPY CODING 💻

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

**Pull request merged**